Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -87,7 +87,8 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
 
     @objc func universalLinkFlow(_ sender: UIButton) {
         // TODO: implement in a future PR - used here so we don't have to remove lazy instantiation
-        payPalClient.tokenize(BTPayPalVaultRequest()) { _, _ in }
+        let request = BTPayPalVaultRequest(enablePayPalAppSwitch: true)
```

Comment:
What do you think if we add a switch (UI) component? and pass its value here, it could help the trader test different scenarios

Reply:
That's the approach we take on Android but on iOS we typically have a separate button for each flow (screenshot below). We can certainly explore moving this to our settings in the future, but we'd likely want to move other aspects as well - this could be a dope open dev project too! Agree this would be a nice to have.
![Simulator Screenshot - iPhone 15 - 2024-03-20 at 14 23 13](https://github.com/braintree/braintree_ios/assets/20733831/191e3c6e-03d3-4004-add1-79faa713a9f1)